mmv1/products/kms/terraform.yaml

```diff
@@ -98,6 +98,24 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       decoder: templates/terraform/decoders/kms.go.erb
       encoder: templates/terraform/encoders/kms_crypto_key.go.erb
       update_encoder: templates/terraform/update_encoder/kms_crypto_key.go.erb
+  CryptoKeyVersion: !ruby/object:Overrides::Terraform::ResourceOverride
+    description: |
+      {{description}}
+
+      Destroying a cryptoKeyVersion will not delete the resource from the project.
+    id_format: "{{crypto_key}}"
+    import_format: ["{{crypto_key}}"]
```

Comment:
ideally import format would use `{{crypto_key}}/cryptoKeyVersions/{{version}}` - the problem is there isn't a version field.

Even if there were a version field, you'd need a `custom_code.custom_import` override because `crypto_key` can contain slashes, and those aren't supported by the current default import implementation.

I'd recommend looking into a custom import, but there will also likely be additional tweaking needed to get this to work as expected.

Reply:
Currently working on it.

Reply:
There is an additional "/" that is causing a lot of problems. 
If the self link is set to {{name}}, the name=1 or 2.
If the self link is set to /{{name}}, the name = GET /v1//projects/tf-test-2080558503729018809/locations/us-central1/keyRings/tf-test-ktx7seecst/cryptoKeys/tf-test-apwjw3cwpe/cryptoKeyVersions/1?alt=json HTTP/1.1

Reply:
I did a big workaround for the custom_import to solve the problem above.